IBuildaAZoo/IBuildaAZoo/Cow.cs

```diff
@@ -3,8 +3,13 @@ namespace IBuildaAZoo
 {
     public class Cow
```

Comment:
This should inherit from `Animal`, i.e.

```suggestion
    public class Cow : FarmAnimal
```